assets/css/utilities/_borders.scss

```diff
@@ -13,6 +13,12 @@
   }
 }
 
+.border-desktop {
+  @media (min-width: 767px) {
+    border: 1px solid $border-grey;
+  }
+}
+
```

Comment:
I also moved borders to https://github.com/blockchain/brand-assets/blob/css/sass/utilities/_borders.scss. Do you think this is wallet specific? It's fine to keep it in My-Wallet-V3-Frontend only, but we'll need to add https://github.com/blockchain/My-Wallet-V3-Frontend/pull/1036/files#diff-ce9ddbc6932bc1ccd28e79c4bf927f8fL11 back.